### PR TITLE
XL in grid and setInnerText into MaterialIcon

### PR DESCRIPTION
**Pull request by paulux84**
- added metodo setInnerText to MaterialIcon so we can use Google Icons not yet into IconType
- added XL type (@media min-width 1600)

### CODE REVIEW

**Comment by BenDol** [comment]
We shouldn't modify the materializecss css file. Can you add media type to the overridecss.css file?

**Comment by gilberto-torrezan** [comment]
@BenDol That was a doubt I had when needed to change CSS files on GWT-Material. Can you summarize (maybe at the [Contributing](https://github.com/GwtMaterialDesign/gwt-material/wiki/Contributing) page) which files shouldn't be touched and which ones can be modified when contributing to the project?

**Comment by paulux84** [comment]
@BenDol i can obviously, but...
...this is the code part added...

'''
@media only screen and (min-width: 1600px) {
    .row .col.xl1 {
        width: 8.33333%;
        margin-left: 0;
    }
    .row .col.xl2 {
        width: 16.66667%;
        margin-left: 0;
    }
    .row .col.xl3 {
        width: 25%;
        margin-left: 0;
    }
    .row .col.xl4 {
        width: 33.33333%;
        margin-left: 0;
    }
    .row .col.xl5 {
        width: 41.66667%;
        margin-left: 0;
    }
    .row .col.xl6 {
        width: 50%;
        margin-left: 0;
    }
    .row .col.xl7 {
        width: 58.33333%;
        margin-left: 0;
    }
    .row .col.xl8 {
        width: 66.66667%;
        margin-left: 0; }
    .row .col.xl9 {
        width: 75%;
        margin-left: 0; }
    .row .col.xl10 {
        width: 83.33333%;
        margin-left: 0; }
    .row .col.xl11 {
        width: 91.66667%;
        margin-left: 0; }
    .row .col.xl12 {
        width: 100%;
        margin-left: 0; }
    .row .col.offset-xl1 {
        margin-left: 8.33333%; }
    .row .col.offset-xl2 {
        margin-left: 16.66667%; }
    .row .col.offset-xl3 {
        margin-left: 25%; }
    .row .col.offset-xl4 {
        margin-left: 33.33333%; }
    .row .col.offset-xl5 {
        margin-left: 41.66667%; }
    .row .col.offset-xl6 {
        margin-left: 50%; }
    .row .col.offset-xl7 {
        margin-left: 58.33333%; }
    .row .col.offset-xl8 {
        margin-left: 66.66667%; }
    .row .col.offset-xl9 {
        margin-left: 75%; }
    .row .col.offset-xl10 {
        margin-left: 83.33333%; }
    .row .col.offset-xl11 {
        margin-left: 91.66667%; }
    .row .col.offset-xl12 {
        margin-left: 100%; } }
'''

overridecss is no correct place for add this...
Add you this part to overridecss...i simply remove it from this pullrequest, ok?

**Comment by kevzlou7979** [comment]
@paulux84 All css styles that overrides Materializecss will be coded inside overridecss.css file. We are not going to touch the core files (Materializecss) because everytime they release we update also the previous core files.

**Comment by paulux84** [comment]
ok, done...
